cmd/format-disk-cache.go

```diff
@@ -49,6 +51,9 @@ type formatCacheV1 struct {
 		// Disks field carries the input disk order generated the first
 		// time when fresh disks were supplied.
 		Disks []string `json:"disks"`
+		// Distribution algorithm represents the hashing algorithm
+		// to pick the right set index for an object.
+		DistributionAlgo string `json:"distributionAlgo"`
```

Comment:
this is not used anywhere can we use it? you can use `hashKey`

Reply:
yes we should use it using `hashKey`, in this PR I just wanted to fix the backend format quickly